.orchestra/config/components/toolchain/lib/gcc.lib.yml

```diff
@@ -9,11 +9,11 @@
 
 #@yaml/text-templated-strings
 ---
-#@ def gcc_dependencies(stage, toolchain_name, musl_version, uclibc_version, linux_version, mingw64_version):
+#@ def gcc_dependencies(stage, toolchain_name, musl_version, uclibc_version, linux_version, mingw64_version, custom_pkgconfig):
 #@ if stage == 1:
-#@   build = "~headers"
+#@   suffix = "-headers"
 #@ elif stage == 2:
-#@   build = ""
+#@   suffix = "-full"
```

Comment:
`-full` is misleading, since it seems to be a superset of `-headers` and `-nolibs`, while it's not.
What about `-headers`, `-base` and `-extra-libraries`?

Reply:
In fairness the issue is mitigated by the fact that `-full` depends on `-nolibs` which depends on `-headers`, so you can't install only `-full`. 
One might find `-extra-libraries` misleading and specify a dependency on `-base`, which does not provide the essential `libpthread`, but ok, I'll make the change to mingw and the other libc-s.

Reply:
Yes, but when you uninstall it you're removing only parts of the libraries.
If you prefer you can specify all three of them as dependencies where needed or make a meta-component that includes all of them.
But `-full` doesn't seem an appropriate representation of what the component installs.